lib/perl/Genome/Config.t

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env genome-perl
+
+use strict;
+use warnings;
+
+use Genome::Test::Config qw(setup_config);
+
+use Test::More tests => 6;
+use Test::Fatal qw(exception);
+
+
+subtest 'Genome does not need to be loaded for Genome::Config' => sub {
+    use_ok('Genome::Config');
+    ok(!scalar(grep { $_ eq 'Genome.pm' } keys %INC), 'Genome has not been loaded');
+};
+
+subtest 'basic lookup' => sub {
+    plan tests => 2;
+
+    my ($temp_dirs, $new_temp_dir) = Genome::Test::Config::temp_dir_helper();
```

Comment:
I think you are describing what I wanted.  I want this scope/`subtest` to hold the (only) reference to the `File::Temp` object so that it is cleaned up when the `subtest` ends.